tests/graph/test_osm_network_wrapper.py

```diff
@@ -0,0 +1,222 @@
+import shutil
+from pathlib import Path
+
+import networkx as nx
+import pytest
+from networkx import Graph, MultiDiGraph
+from networkx.testing import assert_graphs_equal
+from networkx.utils import graphs_equal
+from shapely.geometry import LineString, Polygon
+from shapely.geometry.base import BaseGeometry
+
+from tests import test_data, slow_test
+import ra2ce.graph.networks_utils as nut
+from ra2ce.graph.osm_network_wrapper import OsmNetworkWrapper
+
+
+class TestOsmNetworkWrapper:
+    @pytest.fixture
+    def _config_fixture(self) -> dict:
+        _test_dir = test_data / "graph" / "test_osm_network_wrapper"
+
+        yield {
+            "static": _test_dir / "static",
+            "network": {
+                "polygon": "_test_polygon.geojson"
+            },
+            "origins_destinations": {
+                "origins": None,
+                "destinations": None,
+                "origins_names": None,
+                "destinations_names": None,
+                "id_name_origin_destination": None,
+                "category": "dummy_category",
+                "region": "",
+            },
+            "cleanup": {"snapping_threshold": None, "segmentation_length": None},
+        }
+
+    @pytest.mark.parametrize("config", [
+        pytest.param(None, id="NONE as dictionary"),
+        pytest.param({}, id="Empty dictionary"),
+        pytest.param({"network": {}}, id='"Empty Config["network"]"'),
+        pytest.param({"network": "string"}, id='"invalid Config["network"] type"')
+    ])
```

Comment:
Just say in the id , dictionary with network key and no value, dictionary with network key and value